net/tsdial/tsdial.go

```diff
@@ -76,6 +75,7 @@ type Dialer struct {
 	dnsCache         *dnscache.MessageCache // nil until first non-empty SetExitDNSDoH
 	nextSysConnID    int
 	activeSysConns   map[int]net.Conn // active connections not yet closed
+	routes           art.Table[bool]  // true indicates routes that point into the Tailscale interface
```

Comment:
Prefer to use bart.Table these days as it's more memory efficient

Reply:
Updated to use `bart.Table` instead of `art.Table`. Thanks!